### ISSUE TITLE
Typos in settings page

### ISSUE THREAD

**Issue by papabak** [music-assistant/hass-music-assistant#1932]
### What version of Music Assistant has the issue?

2.9.0b100

### The problem

Typos in the Dutch settings core page. “Artietsen” “biblthioteek”

![IMG_9201](https://github.com/music-assistant/hass-music-assistant/assets/97854063/6903c33f-417c-4e1a-b04d-55c08337fcb4)

### How to reproduce

Is present in several versions 

### What version of Home Assistant Core are your running

2024.2.2

### What type of installation are you running?

Home Assistant OS

### On what type of hardware are you running?

ODROID

**Comment by OzGav** [comment]
It would be great if you could sign up to lokalise and fix this yourself. See the docs,

**Comment by papabak** [comment]
Hi OzGav,I would certainly like to give this assistance. I read the docs, but I’m not sure if I looked in the right place as I didn’t find the option to join and make the corrections myself.Can you please point me to the proper place?Best regards,DirkVerstuurd vanaf mijn mobiele deviceOp 20 feb 2024 om 09:42 heeft OzGav ***@***.***> het volgende geschreven:﻿
It would be great if you could sign up to lokalise and fix this yourself. See the docs,

**Comment by OzGav** [comment]
https://music-assistant.io/help/lokalise/

**Comment by OzGav** [comment]
Have you been able to resolve this?

**Comment by papabak** [comment]
I found the right place yesterday. On my phone it is not comfortable, so will work on it from my PC today.
________________________________
Van: OzGav ***@***.***>
Verzonden: Thursday, February 22, 2024 5:28:48 AM
Aan: music-assistant/hass-music-assistant ***@***.***>
CC: papabak ***@***.***>; Author ***@***.***>
Onderwerp: Re: [music-assistant/hass-music-assistant] Typos in settings page (Issue #1932)


Have you been able to resolve this?

**Comment by OzGav** [comment]
Just checking how you are going with this as we would like to close it if possible

**Comment by papabak** [comment]
I was a few days in France. Do it tomorrow.

Kind regards,
Dirk
________________________________
Van: OzGav ***@***.***>
Verzonden: Sunday, February 25, 2024 2:49:15 PM
Aan: music-assistant/hass-music-assistant ***@***.***>
CC: papabak ***@***.***>; Author ***@***.***>
Onderwerp: Re: [music-assistant/hass-music-assistant] Typos in settings page (Issue #1932)


Just checking how you are going with this as we would like to close it if possible

**Comment by OzGav** [comment]
Hi Dirk,

Just checking if you have completed this so I can close this report.

Cheers!

**Comment by papabak** [comment]
Hi Ozgav,

Yesterday I did some translations from English to Dutch. However I couldn’t
find previous translated parts containing the misspellings (Music Assistant
settings Core). Seems only untranslated parts were offered. I will have a
second look to see if I missed something yesterday.

Best regards,
Dirk

Op vr 1 mrt 2024 om 02:06 schreef OzGav ***@***.***>

**Comment by OzGav** [comment]
Thanks. If you can't sort it out I will ask the translations gurus what the situation is

**Comment by papabak** [comment]
Hi OzGav,

I have translated all remaining (26) parts to Dutch. I guess these texts will be veriefied by a second pair of eyes. However, the texts containing the misspellings where not among the texts offered as these where untranslated texts only.

I  like to hear if this was what was expected and how I can make the corrections to the misspelling in MA Setting Core texts.

Best regards,
Dirk

**Comment by OzGav** [comment]
@jozefKruszynski how do existing translations get updated?

**Comment by jozefKruszynski** [comment]
Whoever the contributors to the languages are must go to Lokalise and update the translations there. Then we can trigger a PR from Lokalise to the GitHub repo

**Comment by OzGav** [comment]
> Whoever the contributors to the languages are must go to Lokalise and update the translations there. Then we can trigger a PR from Lokalise to the GitHub repo

Do you mean only the original contributor can update their sentences?

**Comment by jozefKruszynski** [comment]
Not at all, actually anyone who wants to sign up to contribute can do so.

**Comment by OzGav** [comment]
@jozefKruszynski @papabak can’t seem to find the original sentences to fix

**Comment by jozefKruszynski** [comment]
Ah, this could be because the original upload was with all the existing languages which included dutch.
I'll try to take a look a little later on

**Comment by OzGav** [comment]
Any luck @jozefKruszynski ?

**Comment by jozefKruszynski** [comment]
Just checked and that string is easily findable in lokalise via search. Not sure why there is an issue finding it.

**Comment by OzGav** [comment]
@papabak please see above

**Comment by OzGav** [comment]
@papabak we will close this soon and request you make the changes on lokalise

**Comment by OzGav** [comment]
I have made the changes to the initial report. Plenty more translations are required but this should be fixed on the next import
